src/btree.c

```diff
@@ -3317,8 +3324,8 @@ btree_verify_node(cache        *cc,
                                   btree_get_tuple_key(cfg, node.hdr, idx + 1))
                 >= 0)
             {
-               platform_log_stream("out of order tuples\n");
-               platform_log_stream("addr: %lu idx %2u\n", node.addr, idx);
+               platform_error_log("out of order tuples\n");
```

Comment:
This change is **very nice**! Along with L3310 which deletes `platform_open_log_stream()`, this change makes the code much clear and readable that the output messages are error indicators, going to an output error-log stream.

So, thanks!